src/index.js

```diff
@@ -4,12 +4,21 @@ import CssBaseline from '@material-ui/core/CssBaseline';
 import { ThemeProvider } from '@material-ui/core/styles';
 import App from './App';
 import theme from './theme';
+import { createStore } from 'redux';
+import allReducers from './redux/reducers';
+import { Provider } from 'react-redux';
 
+const store = createStore(
+	allReducers,
+	window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
This line would cause the app to crash unless you have installed a react extension into your browser. Use this library instead `redux-devtools-extension`

Reply:
@AJAkimana , I have applied your feedback. Please check.